test/plugin_update_command.bats

```diff
@@ -3,32 +3,39 @@
 load test_helpers
 
 setup() {
-  BASE_DIR=$(mktemp -dt asdf.XXXX)
-  HOME=$BASE_DIR/home
-  ASDF_DIR=$HOME/.asdf
-  git clone -o local "$(dirname "$BATS_TEST_DIRNAME")" "$ASDF_DIR"
-  git --git-dir "$ASDF_DIR/.git" remote add origin https://github.com/asdf-vm/asdf.git
-  mkdir -p "$ASDF_DIR/plugins"
-  ASDF_BIN="$ASDF_DIR/bin"
-
-  # shellcheck disable=SC2031
-  PATH=$ASDF_BIN:$ASDF_DIR/shims:$PATH
-  install_dummy_plugin
-
-  PROJECT_DIR=$HOME/project
-  mkdir $PROJECT_DIR
```

Comment:
I believe this code was to set the test env asdf root to have a `.git` dir with the remote set to https://github.com/asdf-vm/asdf.git to enable testing of plugin repo fetching.

I need to put this back in